src/lib/ReflexContainer.js

```diff
@@ -369,6 +369,8 @@ export default class ReflexContainer extends React.Component {
           this.computeAvailableOffset(
             splitterIdx, dir * offset)
 
+        this.state.isSetPropSize = data.isSetPropSize
```

Comment:
this isn't a proper react way to alter the state. if `isSetPropSize` isn't used for rendering, then just a separate variable could be used instead.

Reply:
I corrected the thing that you pointed out.
Thanks.